### PR TITLE
Bg user story 49

### LINKED ISSUE #52: User Story 49, Merchant sees an order show page
As a merchant employee
When I visit an order show page from my dashboard
I see the recipients name and address that was used to create this order
I only see the items in the order that are being purchased from my merchant
I do not see any items in the order being purchased from other merchants
For each item, I see the following information:
- the name of the item, which is a link to my item's show page
- an image of the item
- my price for the item
- the quantity the user wants to purchase

### PR DESCRIPTION
**Pull request by brian-greeson**
## Merchant can view order show page
Closes: #52 
- Added merchant order show page and tests
- Added ItemOrder model method `blongs_to_merchant_id?` methods
- Moved common actions in merchant namespace to base controller

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -6,23 +6,25 @@
...
-    patch '/merchants/:merchant_id', to: "merchants#update"
+    get "/dashboard", to: "dashboard#index"
+    get "/users", to: "users#index"
+    get "/merchants", to: "merchants#index"
+    get "/merchants/:merchant_id", to: "merchants#show"
+    patch "/merchants/:merchant_id", to: "merchants#update"
     patch "/order/:id", to: "order#ship", as: :order_ship
     namespace :merchant do
-      get ':id/items', to: "items#index", as: :items
+      get ":id/items", to: "items#index", as: :items
```
Comment (kelshadarby) [inline config/routes.rb:16]: Good catch!

File: spec/models/item_orders_spec.rb [hunk 2]
```diff
@@ -58,6 +58,13 @@
       expect(item_order_2.unfulfilled?).to eq(false)
     end
 
+    it 'belongs_to_merchant?' do
+      todd = Merchant.create(name: "Todds", address: '123 TOdd Rd.', city: 'Denver', state: 'CO', zip: 80203)
```
Comment (kelshadarby) [inline spec/models/item_orders_spec.rb:62]: My favorite merchant we've created so far

**Review by kelshadarby — APPROVED** [review]

File: app/views/merchant/orders/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,48 @@
+<section class = "shipping-address">
+  <h1 align = "center">Shipping Info</h1>
+  <table>
+    <tr>
+      <th>Name</th>
+      <th>Address</th>
```
Comment (fredrondina) [inline app/views/merchant/orders/show.html.erb:6]: very clean, nice use of the table

**Review by fredrondina — APPROVED** [review]
great work again

**Pull request merged**